miasm2/arch/ppc/sem.py

```diff
@@ -294,10 +294,10 @@ def mn_do_lswi(ir, instr, rd, ra, nb):
     if nb == 0:
         nb = 32
     i = 32
-    raise "%r not implemented" % instr
+    raise NotImplementedError("%r not implemented" % instr)
```

Comment:
Its better to raise a `RuntimeError` (as we do in others miasm code), because `NotImplementedError` is intended to be used for abstract method